test/format/format.sh

```diff
@@ -48,9 +48,6 @@ smoke_list=(
 	"$smoke_base_1 file_type=row huffman_key=1 huffman_value=1"
 	"$smoke_base_1 file_type=var huffman_key=1 huffman_value=1"
 
-	# Abort/recovery test.
-	"$smoke_base_1 file_type=row abort=1"
-
```

Comment:
Any reason to remove the abort/recovery test?

Reply:
Refer to discussion in WT-5322. Keith suggested removing abort/recovery as it will use the some empty page trick that doesn't work well with AddressSanitizer.

Reply:
Oh thanks. Looks I missed the discussion on that ticket.